src/test/java/com/kainos/ea/controller/JobRoleControllerTest.java

```diff
@@ -0,0 +1,63 @@
+package com.kainos.ea.controller;
+
+import org.example.controllers.JobRoleController;
+import org.example.daos.JobRoleDao;
+import org.example.models.JobRoleResponse;
+import org.example.services.JobRoleService;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mockito;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import javax.ws.rs.core.Response;
+import java.sql.Date;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+@ExtendWith(MockitoExtension.class)
+public class JobRoleControllerTest {
+
+    JobRoleService jobRoleService = Mockito.mock(JobRoleService.class);
+
+    JobRoleController jobRoleController = new JobRoleController(jobRoleService);
+
+    @Test
+    public void getJobRoles_shouldReturnResponseCode200WithJobRoleResponses() throws SQLException {
```

Comment:
picky i know, but from a naming POV it should be something like 

getJobRoles_shouldReturnResponseCode200WithJobRoleResponses_whenNoErrorsThrown

modify all tests so that this naming convention is observed